Description: remove the bundler dependency
 Prevent bundler from handling the gems.
 .
 ruby-responders (2.0.2-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #nnnn)
Author: Balasankar C <balasankarc@autistici.org>


--- ruby-responders-2.0.2.orig/test/test_helper.rb
+++ ruby-responders-2.0.2/test/test_helper.rb
@@ -1,4 +1,4 @@
-require 'bundler/setup'
+#require 'bundler/setup'
 require 'minitest/autorun'
 require 'mocha/setup'
 
